imagetest/utils/test_utils.go

```diff
@@ -104,3 +105,14 @@ func ExtractBaseImageName(image string) (string, error) {
 	imageName := strings.Join(splits[:len(splits)-1], "-")
 	return imageName, nil
 }
+
+// GenerateSSHKeyPair generate ssh key pair.
+func GenerateSSHKeyPair(user string) error {
+	email := fmt.Sprintf("%s@test.com", user)
```

Comment:
The canonical fake domain is example.com - we should avoid using other domain names that may exist.